frontend/src/components/shared/ImpactReportField.js

```diff
@@ -253,18 +276,15 @@ export default function ImpactReportField({defaultValue, required, onChange, lab
                 }
                 error={!!errors.forStakeholderOutcome}
                 helperText={errors.forStakeholderOutcome}
-                required={required}
-                // onBlur={() => {
-                //   if (state.forStakeholderOutcome) {
-                //     setErrors(errors => ({...errors, forStakeholderOutcome: null}));
-                //   }
-                // }
-                // }
+                required={isFieldRequired(attriConfig, attribute2Compass, 'stakeholdersoutcome')}
+                onBlur={validateField(form, attriConfig, 'stakeholdersoutcome', attriConfig['stakeholdersoutcome'], setErrors)}
```

Comment:
Should be 'stakeholderOutcomes' instead of 'stakeholdersoutcome', please make sure the key you passed to the function is same with the key in attribute2Compass. 

Also, instead of attriConfig['stakeholdersoutcome'], it should be attribute2Compass['stakeholdersoutcome'].
Sorry for misleading you in some of the files I did, I also made mistakes on some of code, I will fix them soon.